frontend/packages/kubevirt-plugin/integration-tests/tests/models/rhvImportWizard.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-await-in-loop */
+import { browser, ExpectedConditions as until } from 'protractor';
+// import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { click, fillInput } from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+import { selectOptionByText, setCheckboxState } from '../utils/utils';
+import {
+  IMPORT_WIZARD_CONN_TO_NEW_INSTANCE,
+  PAGE_LOAD_TIMEOUT_SECS,
+  RHV_WIZARD_CREATE_SUCCESS,
+} from '../utils/consts';
+import * as view from '../../views/importWizard.view';
+import * as rhvView from '../../views/rhvImportWizard.view';
+import { InstanceConfig, rhvConfig, VMImportConfig } from '../utils/types';
+import { VirtualMachine } from './virtualMachine';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import * as wizardView from '../../views/wizard.view';
+import { ImportWizard } from './importWizard';
+
+export class RhvImportWizard extends ImportWizard {
```

Comment:
it could be made more generic, but looks pretty good. We can do more refactoring in a followup